### PR TITLE
move nightly coverage to inseng

### PR DESCRIPTION
**Pull request by cobbal**
refs: MBL-14546
affects: none
release note: none

[run-nightly]

### CODE REVIEW

File: scripts/coverage/coverage.js [hunk 1]
```diff
@@ -305,7 +305,7 @@ function syncCoverage () {
     return
   }
   const s3folder = scheme.toLowerCase()
-  if (process.env.BITRISE_GIT_BRANCH == "master") { // This is a master run, push to s3
+  if (process.env.BITRISE_GIT_BRANCH == "master" || true) { // This is a master run, push to s3
```
Comment (cobbal) [inline scripts/coverage/coverage.js:308]: Temporary for testing

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/nryi5h5) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fnryi5h5)
teacher: [Link](https://rebrand.ly/2o1f2l6) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F2o1f2l6)
parent: [Link](https://rebrand.ly/c08d78e) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fc08d78e)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.js`.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


## Error Error
```
Cannot find module './citests/coverage-summary-master.json'
Require stack:
- /Users/vagrant/git/scripts/coverage/dangerfile-utils.js
- dangerfile.js
- /Users/vagrant/git/node_modules/danger/distribution/runner/runners/inline.js
- /Users/vagrant/git/node_modules/danger/distribution/commands/danger-runner.js
Error: Cannot find module './citests/coverage-summary-master.json'
Require stack:
- /Users/vagrant/git/scripts/coverage/dangerfile-utils.js
- dangerfile.js
- /Users/vagrant/git/node_modules/danger/distribution/runner/runners/inline.js
- /Users/vagrant/git/node_modules/danger/distribution/commands/danger-runner.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at checkCoverage (/Users/vagrant/git/scripts/coverage/dangerfile-utils.js:21:18)
    at Object.<anonymous> (dangerfile.js:213:5)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at requireFromString (/Users/vagrant/git/node_modules/require-from-string/index.js:28:4)
    at Object.<anonymous> (/Users/vagrant/git/node_modules/danger/distribution/runner/runners/inline.js:134:34)
    at step (/Users/vagrant/git/node_modules/danger/distribution/runner/runners/inline.js:32:23)
```
### Dangerfile
```
------------------^
```
  

[MBL-14546](https://instructure.atlassian.net/browse/MBL-14546)

#### Affected Apps: None
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cobbal** [comment]
See coverage at https://code-coverage.inseng.net/ios/coverage/nightlytests/

**Comment by vanwagonet** [comment]
Do you need me to handle updating the code-coverage index page, or do you have inseng credentials you can use?

**Pull request merged**